source/content/guides/disaster-recovery/03-site-goes-down.md

```diff
@@ -42,23 +42,21 @@ Because incidents are declared when a platform issue meets a minimal downtime or
 
 ## Incident Escalation
 
-Although filing an emergency on-call ticket will escalate your downtime incident within the support team and ensure you receive the fastest response, you may also want to alert your broader Pantheon account team. Depending on the situation, your escalation path may differ.
+Filing an emergency ticket will escalate your downtime incident within the Support team and ensure you receive the fastest response. However, you may also want to alert your broader Pantheon account team. Depending on the situation, your escalation path may vary.
 
 ### Support Channels
 
-* **Ticketing**: If your site is suffering downtime on the Live environment, your first step should be to open a support ticket. Chat normally has a quicker response time, but emergency on-call tickets are absolutely escalated and response times to these tickets should be comparable.
+* **Ticket**: In cases of downtime or significant functional failure in the live environment, the first step is to open a support ticket. While live chat generally offers a quicker response time, emergency tickets (for **Diamond** and **Platinum** customers) are escalated and should have comparable response times.
 
-* **Slack**: Diamond tier accounts can have access to a dedicated Slack channel in which customers can interact directly with their CSM, AM, and primary support resources. This is primarily intended as a means for quick communication and collaboration, and should not be used in lieu of the ticketing system, as there are no SLOs associated with Slack channels.
```

Comment:
Support leadership was the one who recommended this change initially 👍 
I'll review it today for clarity though!

Reply:
Alrighty, thanks for the patience here. I can't speak to the Slack side of things, my comment above was in reference to "moving away from the phrase on-call" in general. 

(Speculatively, no authority here) My 2¢ is that removing Slack as a section shouldn't hurt, since this document defines ways to correctly contact support, while Slack's section is explicitly a "dont use this method section". There are many ways you could try and contact support that we don't accept, this one feels out of place.

Reply:
I do not have insights into this. The changes that we are making to the Support page in the dashboard (Phoebe experience) do not change expectations on what to expect with Slack Community

Reply:
The primary goal, to Dylan's point, is to move away from "on-call" being a part of the user-facing Support page copy. We are not making changes to the types, scope, or ways that users can get assistance

Reply:
This would be for shared Slack channels within Pantheon's official team slack itself. I think we should keep this in, to avoid confusion, since it's not immediately related to the `on-call` to `emergency` change.

Reply:
Got it. Also a +1 for keeping it unless there was specific feedback to remove at this time. I do not know of any plans to change any scope or offering of support/support channels at this time.

Reply:
@dylanbaumann Would be great if you could get clarity from the CSE team. My understanding is that there is no tier-specific Slack channels anymore. But agree this could wait and we focus on the on-call verbiage change in this round of updates.

Reply:
There's definitely channels in Slack where customers have direct access to Pantheon (shared channels, within the Pantheon Staff Slack, where customers are also in their relevant channels too). We should keep it in there for now. I just got pinged on slack by a customer yesterday asking for escalation (which we dont do via Slack), so this is very relevant.

Reply:
This line has been restored, marking convo as resolved